__tests__/app.test.js

```diff
@@ -170,6 +169,62 @@ describe("app", () => {
       });
   });
 
+  describe("POST /api/reviews/:review_id/comments", () => {
+    it("201: responds with posted comment", () => {
+      return request(app)
+        .post("/api/reviews/1/comments")
+        .send({
+          username: "philippaclaire9",
```

Comment:
Lets add another test for a 201 where we test that if we were to send over {
          username: "philippaclaire9",
body: "I am Groot",
banana: "yellow"}

It will still add username and body to the comment but banana will be ignored :D